src/parse.py

```diff
@@ -0,0 +1,135 @@
+import click
+import pandas as pd
+import re
+
+from dataclasses import dataclass
+
+from bonum_logging import log
+
+
+@dataclass
+class CatalogEntry:
+    sheet_filename: str
+    shape: str
+    xls_filename: str
+    sheet_name: str
+
+    def __repr__(self):
+        return (f'{self.__class__.__name__} '
+                f'(sheet_filename={self.sheet_filename}, '
+                f'shape={self.shape}, '
+                f'xls_filename={self.xls_filename},'
+                f'sheet_name={self.sheet_name})')
+
+
+# SmithSystem's Clay produces these files for Bill Schaad
+sharepoint_files = {
+    '1. FY24 Product Sales Slicer.xlsx': ["Inv'd Items Report",
+                                          "Catalog & Item Detail",
+                                          "SEP 2022 Price Book",
+                                          "Inv'd Items FY23",
+                                          "Sheet6",
+                                          "Current Cost"],
```

Comment:
the reasons i added select sheets is b/c a bunch of the sheets have derived data or vslookup's that we will eventually want to treat differently, e.g. like with a table to do lookups.